### PR TITLE
Added VK port-a-log

### PR DESCRIPTION
**Pull request by 9M2PJU**
## What Resources are you wanting to add/modify to the Repo?
-

## Checklist
- [ ] Please make sure that uncommon words are added to the dictionary located in [the wordlist](../config/.wordlist.txt)
- [ ] Does each item have a proper linked reference?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,6 +79,7 @@
 + [Node Red Contesting Dashboard](https://github.com/kylekrieg/Node-Red-Contesting-Dashboard) A basic multi screen contesting dashboard written in Node Red.
 + [TLF](https://github.com/Tlf/tlf) is a logging program for radiosport, or ham radio contests. It supports CQWW, CQ-WPX, ARRL-DX, ARRL-FD, PACC, and EU SPRINT contests as well as many more, and also has general QSO and DXpedition modes.
 + [POLO](https://github.com/ham2k/app-polo) Ham2K Portable Logger is a mobile logging made in React.
++ [VK port-a-log](https://vk3zpf.com/vk-port-a-log) VK port-a-log is an Android logging app primarily for portable logging. It can be used for SOTA, WWFF, Silos On The Air (SiOTA), HEMA, VK Shires or portable operation.
```
Comment (DD5HT) [inline README.md]: Is this open source? :)

Comment (9M2PJU) [reply README.md]: Oh i thought everything can put here. Please delete if it is not an open source.

Comment (DD5HT) [reply README.md]: I prefer to keep it limited to open source software, Thank you. I will update the PR later.

**Comment by 9M2PJU** [comment]
i have created https://github.com/9M2PJU/awesome-hamradio-docker/ 

to collect awesome ham radio docker projects.

**Review by DD5HT — APPROVED** [review]

**Pull request merged**